Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -162,6 +167,9 @@ public override void Update()
 
         private void SetupInput()
         {
+            if (!XRGeneralSettings.Instance.Manager.loaders.Any(l => l is OculusLoader loader && loader.displaySubsystem != null))
```

Comment:
Do we need to do any checks for the other device managers? (whether it be leap motion, windows, etc.)
@keveleigh  @davidkline-ms @CDiaz-MS 

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/7d85b2d46e00ed92af7fdfa21db7a5d0779ba408/Assets/MRTK/Providers/WindowsMixedReality/XRSDK/WindowsMixedRealityDeviceManager.cs#L65-L70

Some place like here looks relevant

Reply:
Added the checks also for WindowsMR - see latest commit.
Also, I added a check to create the controllers for the given loader since MRTK loads the profiles in order. So, if an app is built for Oculus and WindowsMR and the input profile has both device managers, the Oculus controllers are currently created which is not correct (even if the XR SDK might help with the correct inputs).